web/src/components/netflow-table-header.tsx

```diff
@@ -0,0 +1,72 @@
+import * as React from "react";
+import {
+  Thead,
+  Tr,
+  Th,
+  OnSort,
+  SortByDirection,
+} from "@patternfly/react-table";
+import { useTranslation } from "react-i18next";
+
+export enum Columns {
+  date,
+  srcpod,
+  dstpod,
+  srcnamespace,
+  dstnamespace,
+  srcport,
+  dstport,
+  protocol,
+  bytes,
+  packets,
+}
```

Comment:
I think this should be refactored to an interface containing:
- id to access data
- i18n key for translation
- type for table row display

I can be a prop sent from `netflow-traffic.tsx` to the other FunctionComponents since we will need to show / hide columns, change order and so on

Reply:
I get the `id` and `i18n` fields, but how would you do `type` ? Looking at the `NetflowTableRow` component, the way to display varies per column (e.g. date has special formatting, some values are resource links with or without namespace, protocol picks value from an external source...). Or we could also include a `render` function as part of this interface? I mean, it's not impossible, but just to consider if it creates more complexity / more constraints.

Reply:
Yes actually `type` can just be an `enum` to switch on, pointing different render functions in `NetflowTableRow`.

I see the following groups of `type : columns` here :
- formatted date : date
- pod ResourceLink : srcpod / dstpod
- namespace ResourceLink : srcnamespace / dstnamespace
- text : srcport / dstport / protocol / bytes / packets